.circleci/config.yml

```diff
@@ -6,12 +6,17 @@ jobs:
     docker:
       - image: circleci/node:12
     resource_class: large
+    environment:
+      CACHE_VERSION: v1
 
     working_directory: ~/react-spectrum
     steps:
       - checkout
       - restore_cache:
-          key: rsp-yarn-{{ checksum "yarn.lock" }}
+          keys:
+            - rsp-yarn-{{ .Environment.CACHE_VERSION }}-{{ .Branch }}-{{ checksum "yarn.lock" }}
+            - rsp-yarn-{{ .Environment.CACHE_VERSION }}-{{ .Branch }}-
+            - rsp-yarn-{{ .Environment.CACHE_VERSION }}-
```

Comment:
Does this need to restore the `react-spectrum-` cache as well, which is saved below?

Reply:
this will restore `~/.cache/yarn`

`react-spectrum-` key will restore source code, which may be different than what was received from `checkout`
seemed safer to use the checkout code and rebuild what we need

this may be a further optimization down the line if builds start getting longer again, but we'd need to be careful with it

Reply:
hmm so why to we write to the `react-spectrum` cache key then if we don't use it?